Signal/src/contact/OWSContactsManager.m

```diff
@@ -620,6 +699,16 @@ - (UIImage * _Nullable)imageForPhoneIdentifier:(NSString * _Nullable)identifier
     return image;
 }
 
+- (NSComparisonResult (^)(SignalAccount *left, SignalAccount *right))signalAccountComparator
+{
+    return ^NSComparisonResult(SignalAccount *left, SignalAccount *right) {
+        NSString *leftName = [self comparableNameForSignalAccount:left];
+        NSString *rightName = [self comparableNameForSignalAccount:right];
```

Comment:
To be a stable sort, we should sort by uniqueId or recipientId if the display names are the same.

Reply:
cowboy committed in 76481a86a5961521aac19fdcd176a3d45c47f70e